### PR TITLE
change github link to apply link, resolve #50

### LINKED ISSUE #50: change github link to apply link
Github link at the header must transform to a apply link -> we are pushing new kottans to apply, not to use github

### PR DESCRIPTION
**Pull request by suchov**
@mayosh @sudodoki can you look at it pls

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,7 +40,7 @@
       <h1><a href="#intro">Kottans</a></h1>
 
       <div class="suppl-content">
-        <a href="https://github.com/kottans" class="github-link" target="_blank">Github</a>
+        <a href="#contact">Apply</a>
```
Comment (sudodoki) [inline index.html]: where will we have the github-link now?

**Comment by suchov** [comment]
@sudodoki do we really need it? we have a links to http://prntscr.com/aljxk8 developers will find us
maybe better to have CTA for applying?

**Comment by sudodoki** [comment]
@suchov then should add that. Maybe [github-corners](http://tholman.com/github-corners/) ?

**Comment by suchov** [comment]
@sudodoki  sound's great!!! will do soon!

**Pull request closed without merging**